<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutGroupForm</class>
 <widget class="QWidget" name="AboutGroupForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="headLayout">
     <item>
      <layout class="QGridLayout" name="avatarLayout">
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="groupName">
           <property name="text">
            <string>groupname</string>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignBottom">
          <widget class="QLabel" name="subject">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="avatar">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="pixmap">
          <pixmap>../../../img/contact_dark.svg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QGridLayout" name="infoLayout">
     <item row="11" column="0">
      <widget class="QLabel" name="autoAcceptCallLabel">
       <property name="text">
        <string>Auto accept call:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QCheckBox" name="autoacceptfile">
       <property name="accessibleDescription">
        <string>Automatically accept files from contact if set</string>
       </property>
       <property name="text">
        <string>Auto accept files</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="descLabel">
       <property name="text">
        <string>Description:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="desc"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="idLabel">
       <property name="text">
        <string>Id:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="occupants">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="name"/>
     </item>
     <item row="15" column="1">
      <widget class="QPushButton" name="selectSaveDir">
       <property name="text">
        <string>Auto accept for this contact is disabled</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="saveFileLabel">
       <property name="text">
        <string>Default directory to save files:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QComboBox" name="autoacceptcall">
       <item>
        <property name="text">
         <string>Manual</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Audio</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Audio + Video</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Clear history:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="id">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QCheckBox" name="autogroupinvite">
       <property name="accessibleDescription">
        <string>Automatically accept group chat invitations from this contact if set.</string>
       </property>
       <property name="text">
        <string>Auto accept group invites</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="countLabel">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Occupants:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QPushButton" name="removeHistory">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="noticeLabel">
       <property name="text">
        <string>Send notice:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QLineEdit" name="notice"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="aliasLabel">
       <property name="text">
        <string>Alias:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="alias"/>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QGridLayout" name="actionLayout">
     <item row="1" column="0">
      <widget class="QPushButton" name="sendMessage">
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Send message</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QVBoxLayout" name="occupantsLayout">
     <item>
      <layout class="QHBoxLayout" name="friendListLayout"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
